client/src/pages/SignUp/SignUp.tsx

```diff
@@ -1,9 +1,57 @@
 import styles from './SignUp.module.scss';
+import * as React from 'react';
+
+import Button from '@mui/material/Button';
```

Comment:
Use bracket imports for these things, this is the way to import the default export from a specific file like /Button.

You can instead do import { Button, IconButton } from @mui/material. Add anything else you need inside the {} and it will look cleaner.